### ISSUE TITLE
[Bug]: License not detected by GitHub

### ISSUE THREAD

**Issue by SpicySpaceman** [pimcore/pimcore#14334]
### Expected behavior

Not the most important thing, but I guess GitHub should show the correct license.

### Actual behavior

GitHub instead shows "Unknown, Unknown licenses found"

### Steps to reproduce

Have a look at the sidebar of the repo:

![image](https://user-images.githubusercontent.com/10995998/218950812-ae5284ed-7a07-4b44-8855-15eb28650fa8.png)

**Comment by jdreesen** [comment]
Question is if GitHub can detect the license at all, because it's dual licensed and one of it is a custom one.

**Comment by SpicySpaceman** [comment]
> Question is if GitHub can detect the license at all, because it's dual licensed and one of it is a custom one.

Seems like more than one license is indeed possible: https://github.com/rust-lang/rust

But I am unsure how to handle the custom one. This one looks kinda custom to me too and it's just shown as "View License" instead of unknown: https://github.com/python/cpython

Maybe GitHub is confused by "license-faq.md" as the license-xyz pattern is a way to determine between different licenses. But that's just a guess, I am not a GitHub license specialist 😄

**Comment by jdreesen** [comment]
You are right. If you click on it, you can see it detects `license-faq.md` as a license file:
![grafik](https://user-images.githubusercontent.com/424602/218999058-5a989531-c40f-4b49-b9c6-ad19d9d4d189.png)

So what should be done is probably split the licenses in `LICENSE.md` into two distinct files like it is on [rust-lang/rust](https://github.com/rust-lang/rust) and rename the `license-faq.md` to something else not matching Github's pattern.

**Comment by brusch** [comment]
Seems to be obsolete:

<img width="364" height="314" alt="Image" src="https://github.com/user-attachments/assets/634a4541-bd8d-4c39-824a-012b2b0f2efd" />
